react-native-maps/react-native-maps
# fix(ios): wrong coordinates on pressing polyline

### Does any other open PR do the same thing?

No. This PR is essentially the iOS version of #4239

### What issue is this PR fixing?

#4303

### How did you test this PR?

The change affects Apple Maps on iOS.

Test plan:
1. Create a map with a polyline.
2. Press the polyline.
3. The event object should contain the coordinates of the tap location, like the behaviour on Android.
